CalendarApp/BookingRoom/Controllers/SearchController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using BookingRoom.Models;
+using BookingRoom.Models.Displaying;
+
+namespace BookingRoom.Controllers
+{
+    public class SearchController : ApiController
+    {
+        private readonly CalendarConnection _connection;
+
+        public SearchController()
+        {
+            _connection = new CalendarConnection();
+        }
+
+        [HttpGet]
+        public HttpResponseMessage SearchEventById(string id)
+        {
+            var calendars = _connection.GoogleCalendar.CalendarList.List().Execute().Items;
+
+            foreach (var calendarEntry in calendars)
+            {
+                var events = _connection.GoogleCalendar.Events.List(calendarEntry.Id).Execute().Items;
```

Comment:
Is this logic valid? Seems weird to iterate over collection of calendars and then try to get calendar using google api